src/System.Runtime/tests/System/FieldAccessException.cs

```diff
@@ -3,40 +3,38 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
 using Xunit;
 
 public static class FieldAccessExceptionTests
 {
-    internal const int COR_E_METHODACCESS = unchecked((int)0x80131510);
-    internal const int COR_E_FIELDACCESS = unchecked((int)0x80131507);
+    private const int FieldAccessHResult = unchecked((int)0x80131507);
```

Comment:
Please revert this.  The name that was being used matches the native name and should be kept.